src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-role.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Specifies the role that will be used for invoking the Jenkins slaves. If
```

Comment:
Can you rephrase this as follows?

"Specifies the resources role that Jenkins scheduler will receive offers for. If role is set to "*" (default), it will receive offers for unreserved resources. If role is set to something else (e.g., 'jenkins'), it will receive offers for BOTH unreserved resources and 'jenkins' resources."

Thinking about this a bit more, do we want the Jenkins scheduler to receive offers for both unreserved and reserved roles? If only the latter, we will need some logic in JenkinsSheduler to reject unreserved resources.